[2022-07-16 19:45:35,292] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.setup_http_conn manual__2022-07-16T19:45:34.681256+00:00 [queued]>
[2022-07-16 19:45:35,300] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: process_files.setup_http_conn manual__2022-07-16T19:45:34.681256+00:00 [queued]>
[2022-07-16 19:45:35,301] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-16 19:45:35,301] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-16 19:45:35,301] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-16 19:45:35,313] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): setup_http_conn> on 2022-07-16 19:45:34.681256+00:00
[2022-07-16 19:45:35,319] {standard_task_runner.py:52} INFO - Started process 3631 to run task
[2022-07-16 19:45:35,321] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'process_files', 'setup_http_conn', 'manual__2022-07-16T19:45:34.681256+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/process_files.py', '--cfg-path', '/tmp/tmpm8jc6qdz', '--error-file', '/tmp/tmp3vez6hgm']
[2022-07-16 19:45:35,321] {standard_task_runner.py:80} INFO - Job 34: Subtask setup_http_conn
[2022-07-16 19:45:35,368] {task_command.py:371} INFO - Running <TaskInstance: process_files.setup_http_conn manual__2022-07-16T19:45:34.681256+00:00 [running]> on host 98d7be6daedb
[2022-07-16 19:45:35,428] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=process_files
AIRFLOW_CTX_TASK_ID=setup_http_conn
AIRFLOW_CTX_EXECUTION_DATE=2022-07-16T19:45:34.681256+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-16T19:45:34.681256+00:00
[2022-07-16 19:45:35,429] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-16 19:45:35,429] {subprocess.py:74} INFO - Running command: ['bash', '-c', "*** connections delete 'conn_api'\n*** connections delete 'pg_conn'\n\n\n*** connections add 'conn_api' --conn-type 'http' --conn-host 'raw.githubusercontent.com/tadinve/EKG-Foundations/master/04A-postgress/'\n\n\n*** connections add pg_conn \\\n  --conn-type 'postgres' \\\n  --conn-host 'ekg-foundations-postgres-1' \\\n  --conn-login '***' \\\n  --conn-password '***' "]
[2022-07-16 19:45:35,441] {subprocess.py:85} INFO - Output:
[2022-07-16 19:45:37,578] {subprocess.py:92} INFO - Successfully deleted connection with `conn_id`=conn_api
[2022-07-16 19:45:39,998] {subprocess.py:92} INFO - Successfully deleted connection with `conn_id`=pg_conn
[2022-07-16 19:45:43,243] {subprocess.py:92} INFO - [[34m2022-07-16 19:45:43,242[0m] {[34mproviders_manager.py:[0m218} INFO[0m - Optional provider feature disabled when importing '***.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-***-providers-google' package[0m
[2022-07-16 19:45:43,466] {subprocess.py:92} INFO - Successfully added `conn_id`=conn_api : http://:@raw.githubusercontent.com/tadinve/EKG-Foundations/master/04A-postgress/:
[2022-07-16 19:45:47,060] {subprocess.py:92} INFO - [[34m2022-07-16 19:45:47,060[0m] {[34mproviders_manager.py:[0m218} INFO[0m - Optional provider feature disabled when importing '***.providers.google.leveldb.hooks.leveldb.LevelDBHook' from 'apache-***-providers-google' package[0m
[2022-07-16 19:45:47,270] {subprocess.py:92} INFO - [[34m2022-07-16 19:45:47,270[0m] {[34mcrypto.py:[0m82} WARNING[0m - empty cryptography key - values will not be stored encrypted.[0m
[2022-07-16 19:45:47,277] {subprocess.py:92} INFO - Successfully added `conn_id`=pg_conn : postgres://***:******@ekg-foundations-postgres-1:
[2022-07-16 19:45:47,767] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-16 19:45:47,795] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=process_files, task_id=setup_http_conn, execution_date=20220716T194534, start_date=20220716T194535, end_date=20220716T194547
[2022-07-16 19:45:47,840] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-16 19:45:47,870] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
